GameData/RP-0/changelog.cfg

```
@@ -30,7 +29,7 @@ KERBALCHANGELOG
 		versionKSP = 1.12.3
 		CHANGE
 		{
-			change = 
+			change = test
 		}
```

Auto-generated potential fix:
```suggestion
		CHANGE { change = test }
```